api/src/main/java/jakarta/enterprise/inject/build/compatible/spi/BuildServicesResolver.java

```diff
@@ -12,12 +12,13 @@
 
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.Objects;
 import java.util.ServiceConfigurationError;
 import java.util.ServiceLoader;
 import java.util.Set;
 import java.util.TreeSet;
 
-final class BuildServicesResolver {
+public final class BuildServicesResolver {
```

Comment:
perhaps now that this is public it needs type level javadoc

Reply:
Good point.